## **CS123A Module 2 Week 3 Reading**
>### 1. Read the section entitled “INTRODUCTION TO BIOLOGICAL DATABASES” (starting on pg 19).
2 main technologies for DNA sequencing
1. Sanger sequencing (aka dideoxy sequencing, first gen sequencing)
2. Next-generation sequencing (NGS) 
    - allows orders of magnitude more sequencing data to be generated at a fraction of the cost of Sanger sequencing

2 ways of thinking about accessing data
1. terms of individual genes, proteins, or related molecules (singular approach)
    - i.e. information about a specific gene, RNA, and protein can be accessible through data/information
2. large datasets related to a problem of interest (overarching idea)
    - e.g. 
        1. study all variants across all human globin genes
        2. collection of tens of thousands of RNA transcripts to identify a set of regulated transcripts

>### 2. Read the section entitled “CONTENTS OF DNA, RNA, AND PROTEIN DATABASES” (starting on pg 24) up to but not including the section entitled “COMMAND-LINE ACCESS TO DATA AT NCBI” (pg 42).
GenBank
- each sequence is assigned a unique identifier (accession number)
- each sequence is followed by its data file division (e.g. PRI for primate sequences)
- e.g. > looking for sequence of human beta globin
    - keep in mind DNA, RNA-based, and protein sequences are represented in variety of forms
    - e.g. DNA sequence can be represented as a string of nucleotides, a list of codons, or a list of amino acids
    - learn the official name and symbol of gene
        - e.g. human beta globin gene is called HBB
        - RNA or cDNA is generally given same name, protein may differ
- genes are localized to chromosomes so the beta globin gene may be a part of a large fragment of DNA (cosmid, BAC, YAC)
    - BAC: large segment of DNA cloned into bacteria
    - YAC: large segment of DNA cloned into yeast

DNA-Level Data
- DNA-Level Data: Sequence-Tagged Sites (STSs)
    - STSs are short DNA sequences that are unique in the genome
    - STSs are used as landmarks to map genes to chromosomes
    - e.g. HBB gene is mapped to chromosome 11
- DNA-Level Data: Genome Survey Sequences (GSSs)
    - GSSs are short sequences of genomic DNA
    - GSSs are used to identify new genes and to identify genes that are expressed in a particular tissue
    - e.g. GSSs can be used to identify new genes that are expressed in the liver
- DNA-Level Data: High-Throughput Genomic Sequences (HTGs)
    - HTGs are sequences of genomic DNA that are generated by high-throughput sequencing methods
    - HTGs are used to identify new genes and to identify genes that are expressed in a particular tissue
    - e.g. HTGs can be used to identify new genes that are expressed in the liver

RNA-Level Data
- RNA-Level Data: cDNA Databases Corresponding to Expressed Genes
    - cDNA databases contain sequences of cDNA clones
        : cDNA is a DNA copy of an mRNA (complementary DNA)
    - cDNA databases are used to identify new genes and to identify genes that are expressed in a particular tissue
    - e.g. cDNA databases can be used to identify new genes that are expressed in the liver
- RNA-Level Data: Expressed Sequence Tags (ESTs)
    - ESTs are partial DNA sequences of cDNA clones
    - ESTs are used to identify new genes and to identify genes that are expressed in a particular tissue
    - e.g. ESTs can be used to identify new genes that are expressed in the liver
- RNA-Level Data: UniGene
    - UniGene is a database of clusters of ESTs and cDNA sequences that correspond to expressed genes
    - UniGene is used to identify new genes and to identify genes that are expressed in a particular tissue
    - e.g. UniGene can be used to identify new genes that are expressed in the liver
    - UniGene cluster is a db entry for a gene containing a group of corresponding ESTs

Central Bioinformatics Resources: NCBI and EBI
- NCBI: National Center for Biotechnology Information
    - PubMed provides access to 24 million citations in MEDLINE
    - Entrez integrates sceintific literature, DNA, and protein sequence dbs
    - BLAST (Basic Local Alignment Search Tool) is a sequence similarity search tool

European Bioinformatics Institute (EBI)
- EBI is a part of the European Molecular Biology Laboratory (EMBL)

Ensembl 
- Ensembl is a joint project between the EBI and the Wellcome Trust Sanger Institute

RefSeq (Reference Sequence) Project
- goal is to provide the best representative sequence for each normal transcript produced from a gene
- GenBank or RefSeq accession number refers to the most recent version of the sequence

Locus Reference Genomic Project
- goal is to define genomic sequences that can be used as reference standards for genes

Consensus Coding Sequence (CCDS) Project
- goal is to provide a basis for a standard set of gene annotations 

Vertebrate Genome Annotation (VEGA) Project
- goal is to provide high quality, expert annotation abt human and mouse genomes + other vertebrate genomes

Access to Information via Gene Resource at NCBI
- use Gene page to access information like Genomic context (chromosome location), related sequences, etc
- use Nucleotide page for sequence information...

>### 3. Read about the UCSC Genome browser in the section entitled “The University of California, Santa Cruz (UCSC) Genome Browser” (pg 50) in the required textbook.
UCSC Genome Browser
- supports 3 dozen vertebrate and invertabrate genomes
- most widely used genome browser for human and other prominent organisms

>### 4. Do a search for UCSC throughout Chapter 2 in the required textbook and read about what can be done and how search and info can be extracted from the UCSC DB.
#### Search for HBB gene in UCSC Genome Browser example
1. Go to UCSC Bioinformatics site (https://genome.ucsc.edu/)
2. Click on Genome Browser
3. Set clade (group) to Vertebrates, genome to Human, and assembly to any build date
4. Search for HBB gene
5. Click submit and see results
6. Follow RefSeq link to view gene information


